src/spyglass/common/common_position.py

```diff
@@ -267,29 +267,30 @@ def _fix_col_names(spatial_df):
         has_0_idx = all([c in input_cols for c in ZERO_IDX_COLS])
         has_1_idx = all([c in input_cols for c in ONE_IDX_COLS])
 
-        # if unexpected columns, ask user to confirm
-        if len(input_cols) != 4 or not (has_default or has_0_idx or has_1_idx):
-            choice = dj.utils.user_choice(
-                "Unexpected columns in raw position. Assume "
-                + f"{DEFAULT_COLS[:4]}?\n{spatial_df}\n"
-            )
-            if choice.lower() not in ["yes", "y"]:
-                raise ValueError(
-                    f"Unexpected columns in raw position: {input_cols}"
+        if has_default:
+            # move the 4 position columns to front, continue
+            spatial_df = spatial_df[DEFAULT_COLS]
+        elif has_0_idx:
+            # move the 4 position columns to front, rename to default, continue
+            spatial_df = spatial_df[ZERO_IDX_COLS]
+            spatial_df.columns = DEFAULT_COLS
+        elif has_1_idx:
+            # move the 4 position columns to front, rename to default, continue
+            spatial_df = spatial_df[ONE_IDX_COLS]
+            spatial_df.columns = DEFAULT_COLS
```

Comment:
If this works, great, let's go with that, but it's not clear to me which cases this catches that the previous doesn't. The [last time I made edit here](https://github.com/CBroz1/spyglass/commit/c97d8e20fa828cfa745c5893aa014c76b84d8f0a), I thought I could swap these out with the conditional assignment and always rename to defaults, even when `has_default`. 

Or was this change for readability over the conditional assignment? If so, I  would move the `else` up and return early. I don't think we need the current `cols =` on 282 when we already have `input_cols`

Reply:
Essentially there was edge cases caused by the previous assumption that the 1st 4 columns would correspond to the position data if there were extra columns.Can clean up the mentioned redundancy